api/apps/api/src/modules/scenarios/scenario.api.entity.ts

```diff
@@ -94,6 +94,10 @@ export class Scenario extends TimeUserEntityMetadata {
   @Column('uuid', { name: 'project_id' })
   projectId!: string;
 
+  @Column('integer', { name: 'project_scenario_id' })
```

Comment:
I think you may have forgotten to add a migration to this PR's commits

besides adding the column to the table (I'd suggest to use a name that may make it clearer what it's for, as we deal with so many kinds of ids... maybe `project_scenario_numeric_id`?), I recommend to ensure that it is `not null` and unique within a given project it (`alter table scenarios add constraint unique_scenario_numid_per_project unique (project_id, row_count);`)

Reply:
and that max is 999, but I think you have already taken care of this in the migration, given one of the tests below - just add the migration to the PR 😉